sae_analysis/dashboard_runner.py

```diff
@@ -208,8 +216,8 @@ def get_feature_property_df(self):
             / sparse_autoencoder.W_enc.cpu().norm(dim=-1, keepdim=True)
         )
         d_e_projection = cosine_similarity(W_dec_normalized, W_enc_normalized.T)
-        b_dec_projection = sparse_autoencoder.b_dec.cpu() @ W_dec_normalized.T
 
+        assert sparse_autoencoder.cfg.d_sae is not None  # keep pyright happy
```

Comment:
Is `d_sae` really optional? It seems like throughout the code it's assumed to be present